docs/src/pages/customization/css-in-js.md

```diff
@@ -335,3 +335,63 @@ function App() {
 
 export default App;
 ```
+
+## Alternative APIs
+
+Do you think that [higher-order components are the new mixins](https://cdb.reacttraining.com/use-a-render-prop-50de598f11ce)? We don't, but rest assured, because `withStyles()` is a higher-order component.
+It can be extended with **few lines of code** to match different patterns, they are all idiomatic React.
+
+### Render props API (+11 lines)
+
+The term [“render prop”](https://reactjs.org/docs/render-props.html) refers to a simple technique for sharing code between React components using a prop whose value is a function.
+
+```jsx
+const Styled = createStyled({
+  root: {
+    background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
+    borderRadius: 3,
+    border: 0,
+    color: 'white',
+    height: 48,
+    padding: '0 30px',
+    boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .30)',
+  },
+});
+
+function RenderProps() {
+  return (
+    <Styled>
+      {({ classes }) => (
+        <Button className={classes.root}>
+          {'Render props'}
+        </Button>
+      )}
+    </Styled>
+  );
+}
+```
+
+{{"demo": "pages/customization/RenderProps.js"}}
+
+### styled-components API (+13 lines)
+
+styled-components's API removes the mapping between components and styles. Using components as a low-level styling construct can be simpler.
+
+```jsx
+// You can even write CSS with https://github.com/cssinjs/jss-template.
```

Comment:
I like this API, but reading this doesn't really explain to me where this `styled` function coming from?

Reply:
You are right. I adding a comment so people know they can find the implementation in the demo source code 
![capture d ecran 2018-01-22 a 20 30 43](https://user-images.githubusercontent.com/3165635/35240255-24a92390-ffb3-11e7-8580-5a5a37fd34f0.png)